### ISSUE TITLE
Feature request: Bayesian Meta-Analysis robustness check

### ISSUE THREAD

**Issue by tressoldi** [jasp-stats/jasp-issues#800]
Is it possible to add a "robustness check" with graph to the Bayesian Meta-Analysis module?

**Comment by EJWagenmakers** [comment]
I think the reason we did not implement this from the get-go is because it will be very (very) slow. But it could be interesting. Any thoughts, @quentingronau @jstbcs @sophieberkhout ?

**Comment by quentingronau** [comment]
Yes, that would be a nice addition, but one would need to think carefully about how to do this in a way that this is not too slow.

**Comment by tomtomme** [comment]
@tressoldi & @quentingronau 
Are the performance implications still relevant today? Is there maybe a R package that does this more efficiently nowadays?

**Comment by quentingronau** [comment]
You could try the meta_sensitivity function in the metaBMA R package.

**Comment by tressoldi** [comment]
I think a robustness check is always relevant, but I'm aware of the technical limitations you have. Thank you
